middlewares/accesslog/log_appender_test.go

```diff
@@ -0,0 +1,214 @@
+package accesslog
+
+import (
+	"bufio"
+	"compress/gzip"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/containous/traefik/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLogAppenderSimpleFile(t *testing.T) {
+	file := logfilePath("")
+	defer os.Remove(file)
+
+	settings := &types.AccessLog{File: file}
+	la, err := NewLogAppender(settings)
+	assert.Nil(t, err)
+
+	n := 1000
```

Comment:
This code was written by a colleague before he left the team, so I can't say for certain why 1000 was picked.